test/test_nestedtensor.py

```diff
@@ -3084,7 +3084,7 @@ def _validate_nt(self, nt, tensor_list, device, dtype, requires_grad):
         self.assertEqual(nt.values().device, device)
         self.assertEqual(nt.offsets().device, device)
         self.assertEqual(nt.shape[0], batch_size)
-        self.assertTrue(isinstance(nt.shape[1], torch.SymInt))
+        self.assertTrue(isinstance(nt.shape[1], (int, torch.SymInt)))
```

Comment:
Latent bug, because of a ```SymInt``` object's type can be ```int``` in some cases, check the following code: 
https://github.com/pytorch/pytorch/blob/7b99b3efb1009668e0b77e35e9a51206709e11e4/torch/_dynamo/variables/tensor.py#L821-L825

Reply:
Shouldn't you xfail this test under dynamo instead? In eager shape[1] is guaranteed to be SymInt

Reply:
SG!